shell/indicators/StorageIndicator.qml

```diff
@@ -32,68 +32,35 @@ import Hawaii.Hardware 1.0 as Hardware
 import ".."
 
 Indicator {
-    name: "storage"
+    name: qsTr("Storage")
```

Comment:
I didn't see `name` being used anywhere, so I figured we could just use name as the translatable name instead of adding a separate `title` property.